.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+﻿
```

Comment:
I do not think that this directory should be added to source control.

Everybody uses their own IDE with their own options, some people use Visual Studio, some use Visual Studio Code and others use Vim.

Reply:
An accident. It didn't show up in the list of changes when I committed in Visual Studio. Rebased.